batch-nft-reveal/contracts/NFTCollection.sol

```diff
@@ -224,14 +224,13 @@ contract NFTCollection is
         parts[0] = '<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMinYMin meet" viewBox="0 0 350 350">';
         if (_metadataCleared) {
             parts[1] = '<style>.base { fill: white; font-family: serif; font-size: 59px; }</style><rect width="100%" height="100%" fill="black" /><text class="base">';
-            string[7] memory slicedRandomness;
-            string memory randomnessString = Strings.toString(_randomness);
-
-            for (uint8 i = 0; i < 7; i++) {
-                string memory partialNumber = _substring(randomnessString,i*11,(i+1)*11);
+            string[6] memory slicedRandomness;
```

Comment:
I think `svgRows` is a  better name for this variable as slicedRandomness is somewhat ambiguous. I know that I named it this way but I just thought of a better variable name.